Workbooks/Forcepoint Cloud Secuirty Gateway workbook.json

```diff
@@ -0,0 +1,158 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Top 5 Web requested Domains with log severity equal to 6 (Medium)\n---"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\n| where TimeGenerated  <= ago(0m)\n| where DeviceVendor == \"Forcepoint CSG\"\n| where DeviceProduct == \"Web\"\n| where LogSeverity == 6\n| where DeviceCustomString2 != \"\"\n| summarize Count=count() by DeviceCustomString2\n| top 5 by Count\n| render piechart",
```

Comment:
There should be at least 4 charts / graphs in the workbook.(Currently there are 3 charts/graphs present)